documentation/manual/database-backends.md

```diff
@@ -8,18 +8,27 @@ Database backends can implement different algorithms that control how a query is
 OSHDBJdbc / OSHDBH2
 -------------------
 
-The [`ODHSBJDBC`](https://docs.ohsome.org/java/oshdb/0.6.4/aggregated/org/heigit/bigspatialdata/oshdb/api/db/OSHDBJdbc.html) backend is often used in the `OSHDBH2` variant, which expects data to be stored in a single H2 database file. A few example OSHDB extracts in the H2 format are available as download from [downloads.ohsome.org](https://downloads.ohsome.org/OSHDB/v0.6/). It is also possible to generate an extract using an OSM history dump file using the [`oshdb-tool/etl`](https://github.com/GIScience/oshdb/tree/0.6.4/oshdb-tool/etl) module.
+The [`ODHSBJDBC`](https://docs.ohsome.org/java/oshdb/0.7.0/aggregated/org/heigit/ohsome/oshdb/api/db/OSHDBJdbc.html) backend is often used in the `OSHDBH2` variant, which expects data to be stored in a single H2 database file. A few example OSHDB extracts in the H2 format are available as download from [downloads.ohsome.org](https://downloads.ohsome.org/OSHDB/v0.6/). It is also possible to generate an extract using an OSM history dump file using the [`oshdb-tool/etl`](https://github.com/GIScience/oshdb/tree/0.7.0/oshdb-tool/etl) module.
```

Comment:
```suggestion
The [`ODHSBJDBC`](https://docs.ohsome.org/java/oshdb/0.7.0/aggregated/org/heigit/ohsome/oshdb/api/db/OSHDBJdbc.html) backend is often used in the `OSHDBH2` variant, which expects data to be stored in a single H2 database file. A few example OSHDB extracts in the H2 format are available as download from [downloads.ohsome.org](https://downloads.ohsome.org/OSHDB/v0.7/). It is also possible to generate an extract using an OSM history dump file using the [`oshdb-tool/etl`](https://github.com/GIScience/oshdb/tree/0.7.0/oshdb-tool/etl) module.
```

Reply:
same issue in line 4

same issue in line 18 of the first steps readme: https://github.com/GIScience/oshdb/pull/305/files#diff-4fb66faabbdd606820d9cae7b9d8d38f42f61ef915078751554d312a3c560d66L18